unittest/unittest.py

```diff
@@ -200,8 +206,7 @@ def run_class(c, test_result):
             except:
                 print(" FAIL")
                 test_result.failuresNum += 1
-                # Uncomment to investigate failure in detail
-                #raise
+                print_exc()
```

Comment:
would be simpler to `except Exception as e` and then pass e to `print_exc`: no need for `sys.exc_info()[1]`